README.md

```diff
@@ -1 +1 @@
-# about-the-team
\ No newline at end of file
+# about-the-team
```

Comment:
guessing this is a line ending change by git? 🤷

Reply:
Done!